### PR TITLE
Prepare for v3.1.0 release of ceph-csi 

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -84,6 +84,8 @@ build_push_images() {
 
 if [ "${TRAVIS_BRANCH}" == 'master' ]; then
```
Comment (Madhu-1) [inline deploy.sh:85]: remove master from here as we dont need it

File: deploy.sh [hunk 2]
```diff
@@ -84,6 +84,8 @@ build_push_images() {
...
 	export ENV_CSI_IMAGE_VERSION='canary'
+if [ "${TRAVIS_BRANCH}" == 'release-v3.0' ]; then
```
Comment (Madhu-1) [inline deploy.sh:87]: target branch need to be changed to `release-v3.1`

File: deploy.sh (hunk 1 above)
Comment (humblec) [reply deploy.sh]: sure.. done @Madhu-1

File: deploy.sh (hunk 2 above)
Comment (humblec) [reply deploy.sh]: done.

File: .travis.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@ services:
 language: go
 branches:
   only:
-    - master
+    - release-v3.0
```
Comment (nixpanic) [inline .travis.yml:18]: should this not be `release-v3.1` in addition to `master`?

Comment (humblec) [reply .travis.yml]: yeah.. addressed @nixpanic

**Comment by humblec** [comment]
> this neither needs to add the release-3.1 changes, and keep the references to master, or post this against the (non-existing) release-v3.1 branch.

Yep, this is a place holder. As soon as 3.1 is branch off, will target on the same branch instead of master.  @nixpanic

**Comment by Madhu-1** [comment]
@MergifyIO rebase

**Comment by nixpanic** [comment]
Please see #1374 for details about providing the correct version of the image to the CI jobs.

**Comment by Madhu-1** [comment]
@humblec please add the above variable to build.env file

**Comment by Madhu-1** [comment]
Merging manually as mergify rules are not available yet for auto merging

**Comment by humblec** [comment]
Thanks !

**Pull request merged**